npm/javy/README.md

```diff
@@ -0,0 +1,9 @@
+# Javy Lib
+
+This library provides abstractions and convenience methods over Javy's built-in functions.
+
+More documentation TBD>
```

Comment:
Are we missing an opening `<` here? Should we just remove this sentence altogether?